sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationMockTests.cs

```diff
@@ -810,6 +810,11 @@ public async Task AnalyzeOperationWithActionsError()
                         ""code"": ""InvalidRequest"",
                         ""message"": ""Some error"",
                         ""target"": ""#/tasks/sentimentAnalysisTasks/0""
+                      },
```

Comment:
Since we're parsing the service response now, updated one more non-live test we had to include the new API (tests the behavior in case of failures).

Live tests will be added in a following PR.